### ISSUE TITLE
Render hyperlinks in text annotations

### ISSUE THREAD

**Issue by pinussilvestrus** [bpmn-io/bpmn-js#1341]
__Is your feature request related to a problem? Please describe.__

An interesting idea for an extension could be to render hyperlinks added in text annotations to simply direct to external resources, e.g. documentation.

![image](https://user-images.githubusercontent.com/9433996/89895424-0fa54980-dbdc-11ea-94f4-509219fb2884.png)

__Describe the solution you'd like__

Render hyperlinks found in text annotations (or somewhere else?) to make them clickable.

__Describe alternatives you've considered__

/

__Aditional Context__

We identified this issue as a potential candidate for 2020 Hacktoberfest.

**Comment by christyjacob4** [comment]
@pinussilvestrus  Can I take this up as well ?

**Comment by pinussilvestrus** [comment]
Absolutely! Feel free to ask for help if you need some :+1:

**Comment by Hard-Coder05** [comment]
@pinussilvestrus Could you please elaborate this issue? I am not able to understand What is to be done here!

**Comment by pinussilvestrus** [comment]
Sure,

in BPMN diagrams it is possible to create text annotations. Furthermore, this text can include nearly everything, so also URLs. The feature request is about to render those URLs to make them clickable. Currently, you can't click on URLs and direct directly to the page which is referenced.

![Kapture 2020-09-30 at 09 12 25](https://user-images.githubusercontent.com/9433996/94654238-122f4000-02fd-11eb-8b78-135f2b4e18c2.gif)

**Comment by Hard-Coder05** [comment]
okay Thank you understood! Btw The repository dosn't have so many issues for hacktoberfest. Are you going to create new ones? I am a biginner so I was just looking for easy issues to contribute !

**Comment by pinussilvestrus** [comment]
`bpmn-js` is part of the bpmn.io project, so you'll find more issues in other repositories: https://tasks.bpmn.io/board?s=label%3Ahacktoberfest%20

We tried to give a list of easy to start for the hacktoberfest that should also be fun in terms the users are creating real value to our projects. Basically, you can also take other issues, especially when labels as "good first issue" are attached.

**Comment by manishkrjha** [comment]
I'd love to try this. Can you add the links to the files/folders that needed to be changed?

**Comment by botex98** [comment]
@pinussilvestrus Hey :).
I am positive that I can manage this task. May I have it?

**Comment by pinussilvestrus** [comment]
Everyone is free to take this task and open a pull request. We will be happy to review it 👍 Feel free to ask if you need help.

**Comment by botex98** [comment]
@pinussilvestrus are you positive that this issue can be resolved by modifying code in this repository only?

**Comment by pinussilvestrus** [comment]
Maybe it would be better to have this feature as extension. Basically, this should be a deal of the `BpmnRenderer`

**Comment by botex98** [comment]
Yes, but the solution should be as simple as enclosing each link with the `a tag` when drawing the text label?

**Comment by ashish-patwal** [comment]
Hey @pinussilvestrus i would like to work on this issue . Can you guide me to the right files for solving this issue . Also , would be using regex to filter links a good idea to solve this problem ? What do you recommend ?
